packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+
+export interface ButtonCascaderOption {
+  label: string;
+  value: string;
+
+  children?: ButtonCascaderOption[];
+  disabled?: boolean;
+  // Undocumented tooltip API
```

Comment:
Good we have documentation for undocumented things ;) But seriously, what is this property?

Reply:
It's a synonym for `label` https://github.com/react-component/cascader/blob/7d4e8aae7136b8fc8eea1ab55d05461f4d78c908/src/Menus.tsx#L99